clusterloader2/pkg/config/template_provider.go

```diff
@@ -225,7 +225,10 @@ func GetMapping(clusterLoaderConfig *ClusterLoaderConfig) (map[string]interface{
 	if err != nil {
 		return nil, errors.NewErrorList(goerrors.Errorf("mapping creation error: %v", err))
 	}
-	MergeMappings(mapping, envMapping)
+	err = MergeMappings(mapping, envMapping)
+	if err != nil {
+		return nil, errors.NewErrorList(fmt.Errorf("mapping creation error: %v", err))
```

Comment:
Use %w to wrap errors

Reply:
Sorry throughout the clusterloader2 the errors are wrapped with %v, so i want to blend in.